2_base_and_sensor.ipynb

```diff
@@ -73,11 +114,55 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "**For advanced users** To move the move base, we use the `move_base` function defined in the `utils` package. The code is as follows:\n",
```

Comment:
"**For advanced users**: To move the moving base, we use the `move_base` function defined in the `utils` package. The code is as follows:\n",

Notice that I along with editing the text, that I also added a colon ":" after "For advanced users".